rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -743,75 +743,93 @@ bool main_window::InstallPackages(QStringList file_paths, bool from_boot)
 		m_gui_settings->SetValue(gui::fd_install_pkg, file_info.path());
 	}
 
-	if (file_paths.count() == 1 && file_paths.front().endsWith(".pkg", Qt::CaseInsensitive))
+	if (file_paths.count() == 1)
 	{
 		const QString file_path = file_paths.front();
 		const QFileInfo file_info(file_path);
 
-		compat::package_info info = game_compatibility::GetPkgInfo(file_path, m_game_list_frame ? m_game_list_frame->GetGameCompatibility() : nullptr);
-
-		if (!info.is_valid)
+		// #14398 - [Feature request] Install multiple PKG from command line
+		if (file_info.isDir())
 		{
-			QMessageBox::warning(this, tr("Invalid package!"), tr("The selected package is invalid!\n\nPath:\n%0").arg(file_path));
-			return false;
-		}
+			const QDir dir(file_path);
+			const auto dir_file_infos = dir.entryInfoList(QDir::Files);
+			const auto dir_file_infos_count = dir_file_infos.count();
```

Comment:
```suggestion
			gui_log.notice("PKG: Trying to install packages from dir: '%s'", file_path);

			const QDir dir(file_path);
			const auto dir_file_infos = dir.entryInfoList({"*.pkg"}, QDir::Files);

			if (dir_file_infos.empty())
			{
				gui_log.notice("PKG: Could not find any packages in dir: '%s'", file_path);
				return true;
			}

			const auto dir_file_infos_count = dir_file_infos.count();
```

Reply:
Wouldn't filtering by .pkg here potentially stop raps and edats from getting installed afterwards? I'm not sure what lines 837 onwards do though, to be perfectly honest.

Reply:
ah, right, without the filter then.